src/vstart.sh

```diff
@@ -659,6 +659,7 @@ EOF
         bluestore_spdk_mem = 2048"
         else
             BLUESTORE_OPTS="        bluestore block db path = $CEPH_DEV_DIR/osd\$id/block.db.file
+        bluestore block size = 10737418240
```

Comment:
Is this needed?

Reply:
My bad, got accidentally added to this PR. I will remove it.